packages/ws/src/utils/constants.ts

```diff
@@ -0,0 +1,68 @@
+import { readFileSync } from 'node:fs';
+import { join } from 'node:path';
+import { Collection } from '@discordjs/collection';
+import { APIVersion, GatewayOpcodes } from 'discord-api-types/v10';
+import { lazy } from './utils';
+import type { OptionalWebSocketManagerOptions, SessionInfo } from '../ws/WebSocketManager';
+
+/**
+ * Valid encoding types
+ */
+export enum Encoding {
```

Comment:
Is ETF support planned for this PR?

Reply:
No. Me and Vlad have opted against it because erlpack is.. well, awful.

Reply:
Why have an encoding enum then if it's just always going to be JSON?